.github/integration/tests/sda/10_upload_test.sh

```diff
@@ -6,6 +6,20 @@ if [ -z "$STORAGETYPE" ]; then
     exit 1
 fi
 
+if [ "$STORAGETYPE" = "posix" ]; then
+    for t in curl jq openssh-client postgresql-client; do
+        if [ ! "$(command -v $t)" ]; then
+            if [ "$(id -u)" != 0 ]; then
+                echo "$t is missing, unable to install it"
+                exit 1
+            fi
+
+            apt-get -o DPkg::Lock::Timeout=60 update >/dev/null
+            apt-get -o DPkg::Lock::Timeout=60 install -y "$t" >/dev/null
+        fi
+    done
+fi
+
```

Comment:
```suggestion
```

This is already taken care of in a previous step `01_install_dependencies`

Reply:
Correct, I forgot to update this. Fixed now.